vuepress/docs/next/tutorials/devops/plugin-environment-variables.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebarDepth: 2
+---
+
+# Use Plugin Environment Variables to Customize Microservices
+
+This tutorial describes how to use environment variables to customize plugin microservices behavior. Using environment variables as a configuration mechanism allows the same Linux image to run in development and production environments. This is especially useful when managing sensitive information via Kubernetes Secrets.
```

Comment:
"...customize the behavior of plugin microservices" avoids the anticipation of an apostrophe (microservice's or microservices')